### DISCUSSION TITLE
blog/2022/220413-Intranet-penetration-bitcomit/

### DISCUSSION THREAD

**Discussion by giscus** [zhanbao2000/blog#14] [Announcements]
# blog/2022/220413-Intranet-penetration-bitcomit/

Arisa 的个人博客

https://blog.arisa.moe/blog/2022/220413-Intranet-penetration-bitcomit/

**Comment by LuiYang08** [comment]
不明觉厉

**Comment by HysterLc** [comment]
有用，校园网挂梯子下P2P不知道会不会被抓

**Reply by kuyeqiu** [reply]
我在学校四年猛开梯子 没事的

**Reply by LuiYang08** [reply]
啥意思？校园网可以用比特彗星上传做种吗？

**Reply by HysterLc** [reply]
> 啥意思？校园网可以用比特彗星上传做种吗？

校园网自己大概率不行，用梯子的公网做内网穿透做种。
实际上是本机-＞服务器-＞其他人

**Comment by unkmonster** [comment]
你好，如何让比特彗星和peers知道frp服务器的地址呢？

**Reply by unkmonster** [reply]
搞定了，有用
